src/Aspire.Hosting.Azure.AppContainers/api/Aspire.Hosting.Azure.AppContainers.cs

```diff
@@ -46,4 +48,28 @@ public AzureContainerAppCustomizationAnnotation(System.Action<AzureResourceInfra
 
         public System.Action<AzureResourceInfrastructure, global::Azure.Provisioning.AppContainers.ContainerApp> Configure { get { throw null; } }
     }
+}
+
+namespace Aspire.Hosting.Azure.AppContainers
+{
+    public partial class AzureContainerAppEnvironmentResource : AzureProvisioningResource
+    {
+        public AzureContainerAppEnvironmentResource(string name, System.Action<AzureResourceInfrastructure> configureInfrastructure) : base(default!, default!) { }
+
+        public BicepOutputReference ContainerAppDomain { get { throw null; } }
+
+        public BicepOutputReference ContainerAppEnvironmentId { get { throw null; } }
+
+        public BicepOutputReference ContainerAppEnvironmentName { get { throw null; } }
+
+        public BicepOutputReference ContainerRegistryManagedIdentityId { get { throw null; } }
+
+        public BicepOutputReference ContainerRegistryUrl { get { throw null; } }
+
+        public BicepOutputReference LogAnalyticsWorkspaceId { get { throw null; } }
+
+        public BicepOutputReference ManagedIdentityId { get { throw null; } }
+
+        public BicepOutputReference PrincipalName { get { throw null; } }
```

Comment:
I don't think these should be exposed.

Reply:
https://github.com/dotnet/aspire/pull/8529